Room visibility update.md (#46)
vector-im/policies: docs/matrix-org/privacy_notice.md

In unencrypted and encrypted rooms, users connecting to the matrix.org homeserver (directly or over federation) will be able to see messages and files according to the access permissions configuration of the relevant room. This data is stored in the format it was received on our servers, and can be viewed by Element engineers (employees and contractors) under the conditions outlined below.

Rooms have different visibility settings which are determined by the room administrators. The history visibility possibilities are the following, by increasing order of openness (least open first):

> - `joined`: people need to join the room to see the history, and will only see the messages sent after they joined. This behaviour is similar to the experience of IRC on a bouncer, and is how all IRC channels are bridged to Matrix by default

Reviewer: This is just me, but I'd probably drop the IRC bouncer wording. This is on matrix.org, and I think with every passing year fewer and fewer people will know what this means and serve to just conflate the protocols.

> - `joined`: people need to join the room to see the history, and will only see the messages sent after they joined. This behaviour is similar to the experience of IRC on a bouncer, and is how all IRC channels are bridged to Matrix by default

Reviewer: ```suggestion
- `joined`: people can only see messages sent after they were joined, no history is visible before that point. This behaviour is similar to the experience of IRC on a bouncer, and is how all IRC channels are bridged to Matrix by default
```

I just feel the first sentence and the second are a bit confused. Join the room to see the history sounds like `shared`.

> - `joined`: people need to join the room to see the history, and will only see the messages sent after they joined. This behaviour is similar to the experience of IRC on a bouncer, and is how all IRC channels are bridged to Matrix by default

Reviewer: > and is how all IRC channels are bridged to Matrix by default

I'm super worried about this appearing in the privacy doc:

- This is true for one of our bridge implementations.
- This is only true for matrix.org's bridges.
- At the very least we should namedrop the instances this applies to. Otherwise,some rando old IRC bridge might start bridging stuff in world_readable and everyone gets up in arms again.
